templates/server/index.html

```diff
@@ -39,6 +39,29 @@ <h1>Scale out with Ubuntu Server</h1>
   </div>
 </div>
 
+<div class="p-strip is-bordered">
+  <div class="u-fixed-width">
+    <h2>
+      What&rsquo;s new in {{ releases.latest }} LTS
+    </h2>
+    <ul class="p-list is-split">
+      <li class="p-list__item is-ticked">Supported by Canonical until July 2021</li>
+      <li class="p-list__item is-ticked">Runs on all major architectures - x86-64, ARM v7, ARM64, POWER8, POWER9, IBM s390x (LinuxONE) and RISC-V</li>
+      <li class="p-list__item is-ticked">Nftables-based backend for ufw for improved performance and easier administration</li>
+      <li class="p-list__item is-ticked">Support for OVS schema modelling in Netplan for an advanced networking stack configuration</li>
+      <li class="p-list__item is-ticked">Native configuration for WireGuard VPN in Netplan</li>
+      <li class="p-list__item is-ticked">The latest long-term Linux 5.8 kernel for the latest hardware and security updates</li>
+      <li class="p-list__item is-ticked">Updates to QEMU (5.0), libvirt (6.6), PHP (7.4.9), Apache2, GCC (1.189), Python (3.8.6), Bind9 (9.16.6)</li>
```

Comment:
I have asked Rick for the Apache2 version

Reply:
Done